deploy/helm/kuberhealthy/templates/khcheck-pod-status.yaml

```diff
@@ -44,6 +46,37 @@ spec:
     {{- end }}
     serviceAccountName: pod-status-sa
 ---
+{{- if .Values.check.podStatus.cluster.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: jx-pod-status-check-rb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: jx-pod-status-role
+subjects:
+  - kind: ServiceAccount
+    name: jx-pod-status-sa
+    namespace: kuberhealthy
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: jx-pod-status-role
```

Comment:
wondering if we could just keep the names for the cluster rb/cluster role consistent with rb/role ? since only one or the other is applied? so no `jx-` prefix?

Reply:
doh ;) copy / paste - thanks!